src/ripple/shamap/SHAMap.h

```diff
@@ -87,6 +87,8 @@ class SHAMap
     mutable SHAMapState             state_;
     SHAMapType                      type_;
     bool                            backed_ = true; // Map is backed by the database
+    bool                            full_ = false; // Indicates all nodes should reside in a local store.
```

Comment:
How should I think of `full_` in relation to `backed_`?

Reply:
If `backed_` is true, then `full_` determines whether or not every node pertaining to this SHAMap resides in a database.

Reply:
The wording might be confusing. Maybe we can change "full_" to "Map is believed complete in database" or something.